internal/rotate.go

```diff
@@ -196,9 +196,17 @@ func (h *CertRotationHandler) RestartServices() {
 	if err != nil {
 		log.Fatalf("Unable to connect to DBUS for service restarts: %s", err)
 	}
-
-	for _, svc := range []string{"aktualizr-lite.service", "fioconfig.service"} {
+	services := []string{"aktualizr-lite.service", "fioconfig.service"}
```

Comment:
I think we don't need to check the `fioconfig` service/unit status, just the aklite's one needs to be checked.

Reply:
Already changed :+1:

Reply:
If we are going down this rabbit hole. I assume this about people not running the daemon. If you run a custom sota client - I think we need logic for that. We probably need to do #define type thing so this value can be overridden.

It's a bigger piece of work and requires some logic in the yocto recipe. Not sure if we rush to fix this now or wrap up scale test work and then do this. v93 is basically done - so we have a long time to get this fixed.

Reply:
I am not sure this is a rabbit hole. IMO, this is straightforward - just don't restart aklite or custom sota client daemons if their services are not defined or disabled. There are customers that don't set an update agent in a daemon mode or "hold" via the callbacks.

Reply:
k. merge this as is and come back and add support to handle a custom sota client. then update yocto to build this with the correct value. My bigger point was that I think Eric can finish his other task first and then come back to this.

Reply:
I agree on the priority for sure.